### PR TITLE
[JENKINS-39999] Add plugin description to POM.

### PR DESCRIPTION
**Pull request by varyvol**
[JENKINS-39999](https://issues.jenkins-ci.org/browse/JENKINS-39999)

@reviewbybees @andresrc

### CODE REVIEW

**Comment by rsandell** [comment]
Slightly different from the current excerpt on the wiki but :bee:

**Comment by olivergondza** [comment]
:+1:

File: pom.xml [hunk 1]
```diff
@@ -10,6 +10,7 @@
   <packaging>hpi</packaging>
   <version>1.19-SNAPSHOT</version>
   <name>Jenkins Mailer Plugin</name>
+  <description>Allows you to configure email notifications for build results</description>
   
   <url>http://wiki.jenkins-ci.org/display/JENKINS/Mailer</url>
```
Comment (varyvol) [inline pom.xml]: Done.

**Comment by recena** [comment]
🐝

**Comment by andresrc** [comment]
:bee:

@rsandell ok to merge?

**Pull request merged**